draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -851,6 +853,16 @@ To prevent these types of linkability, various methods, including but not limite
 - Use advanced cryptographic schemes, outside the scope of this specification.
 - Issue a batch of SD-JWTs to the Holder to enable the Holder to use a unique SD-JWT per Verifier. This only helps with Verifier/Verifier unlinkability.
 
+Colluding Verifier/Verifier pairs correspond to RP+RP'-U Unlinkability as defined in [@ISO.27551].
```

Comment:
I don't have access to ISO.27551 and am not going to pay for it. I doubt I'm unique there.  `RP+RP'-U` looks odd but what do I know? What value does this sentence bring?

Reply:
no, you are probably not unique. 
one value might be helping readers familiar with ISO 27551 draw the parallel between SD-JWT spec and ISO text, which is not an absolute requirement and I have not seen other IETF specs really do it.

Reply:
I'd suggest the sentence be dropped.

Reply:
```suggestion
```

Reply:
```suggestion
```

Reply:
```suggestion
```